scripts/buildCss.js

```diff
@@ -0,0 +1,41 @@
+// core packages
+const fs = require('fs'),
+      path = require('path')
+
+// npm packages
+const simpleIcons = require('simple-icons')
+
+
+const basePath = path.join(__dirname, '..')
+
+const attributedIcons = Object.values(simpleIcons).map(icon => {
+	return {
+		color: icon.hex,
+		cssClass: icon.title
+			.toLowerCase()
+			.replace(/[ !’]/g, '')
+			.replace(/-/g, '_') // hyphens not supported in ligatures
+			.replace(/\./g, 'dot') // dot not supported in css class names
+			.replace(/\+/g, 'plus') // plus not supported in css class names
+			.replace(/&/g, 'and') // ampersand not supported in css class names
+		,
+		ligature: icon.title
+			.replace(/[ !’]/g, '')
+			.replace(/-/g, '_') // hyphens not supported in ligatures
```

Comment:
It might get a bit confusing that this transformation is(/looks) different from the one in `buildFont.js` it might be better to use `titleToFilename` instead here.

Also, I just noticed that the `titleToFilename` function doesn't work as you're saying here and in [`buildFont.js`](https://github.com/simple-icons/simple-icons-font/blob/83798b8a737ae9e88e73f2b75fc331a6cc21937a/scripts/build.js#L21), as far as I can tell the ligature value does contain `-`. Also, I feel like ligatures do support hyphens, look e.g. at the ligatures in a font like [Fire Code](https://github.com/simple-icons/simple-icons-font/blob/83798b8a737ae9e88e73f2b75fc331a6cc21937a/scripts/build.js#L21).

Reply:
(Statement for the whole replacing topic below)

I remember I tested hyphens with some font tool and that did indeed break it - or it was one of the icon libraries below our stack, but some googling revealed that this may actually just be a problem with Windows or libraries that just straight up deny usage of hyphens *because* Windows can't handle them.

Anyways, I'll put a little research into that and then report back on what we will do.